### DISCUSSION TITLE
Ability to set individual tabs as locked when creating group on startup

### DISCUSSION THREAD

**Discussion by pb0gusz** [indiff/qttabbar#362] [Ideas]
I often find myself closing tabs from my startup group by accident. It would be great addition to make it so there's checkbox to mark them as locked when I open the explorer.

Here's basic mockup how it could look like:

![idea](https://github.com/indiff/qttabbar/assets/69306101/7eb56d1c-fea2-4574-9528-49876b05a076)

**Comment by explor202** [comment]
@indiff I would love to see this same functionality, for the exact same reason!
